pkg/controller/compute/eks.go

```diff
@@ -329,7 +330,7 @@ func (r *Reconciler) _sync(instance *awscomputev1alpha3.EKSCluster, client eks.C
 	instance.Status.SetConditions(runtimev1alpha1.Available(), runtimev1alpha1.ReconcileSuccess())
 	resource.SetBindable(instance)
 
-	// Our cluster is available. Requeue speculative yafter a long wait in case
+	// Our cluster is available. Requeue speculative after a long wait in case
```

Comment:
My guess is that this was originally trying to say "Requeue speculatively after a long wait"